README.MD

```diff
@@ -28,10 +27,19 @@ import { render } from 'react-dom'
 import { createStore, combineReducers } from 'redux';
 
 // 1. Connect your data, similar to react-redux @connect
+// Decorator version
 @asyncConnect({
-  lunch: (params, helpers) => Promise.resolve({id: 1, name: 'Borsch'})
+  lunch: (params, store, helpers) => Promise.resolve({id: 1, name: 'Borsch'})
```

Comment:
@Keats 
> How come the api is different for the decorator and the static method?
Yeah... it's my fault....
i actually thought to change interface significantly:
https://github.com/Rezonans/redux-async-connect/issues/16#issuecomment-178227211
what do you think?